src/AbstractExtendedPdo.php

```diff
@@ -205,7 +205,8 @@ public function errorInfo(): array
      * @see http://php.net/manual/en/pdo.exec.php
      *
      */
-    public function exec(string $statement): int
+    #[\ReturnTypeWillChange]
```

Comment:
Is #[\ReturnTypeWillChange] needed within the 4.x branch, since #[\ReturnTypeWillChange] is needed for PHP >= 8.1 to hide depreciation notices.